lib/setErrorHeaders.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const { kReplyErrorHandlerCalled } = require('./symbols.js')
+
+function setErrorHeaders (error, reply) {
+  var res = reply.raw
+  var statusCode = res.statusCode
+  statusCode = (statusCode >= 400) ? statusCode : 500
+  // treat undefined and null as same
+  if (error != null && reply[kReplyErrorHandlerCalled] === true) {
```

Comment:
How is this treating `undefined` and `null` equivalently?

Reply:
This was here before so I assumed it was true but I just checked and null == undefined -> true.